src/pool/srv_target.c

```diff
@@ -836,6 +887,14 @@ ds_pool_tgt_map_update(struct ds_pool *pool, struct pool_buf *buf,
 		   map != NULL) {
 		struct pool_map *tmp = pool->sp_map;
 
+		rc = pl_map_create_v2(map, &pool->sp_pl_map);
```

Comment:
same, previous sp_pl_map cleanup?

Reply:
Hmm, It looks like you can use pl_map_create() directly to create the placement map inside the hash as the original design, instead of attaching it to pool directly?  Then you do not need pl_map_create_v2() and pl_map_destory_v2()?